api/server/pom.xml

```diff
@@ -32,6 +32,10 @@
     <description>Jakarta WebSocket - Server API</description>
     <url>https://projects.eclipse.org/projects/ee4j.websocket</url>
 
+    <properties>
+        <jpms.name>jakarta.websocket.api</jpms.name>
```

Comment:
I would have expected this to be `jakarta.websocket.server.api` to match (`jakarta.websocket.client.api` in other jar).

But then again, this API isn't well suited for that kind of distinction (why was server ever based on extending client??)